src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,149 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.InvalidRegistrationDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    public static final String LOGIN_INITIAL = "ksdsljkdflkasjd";
+    public static final String PASSWORD_INITIAL = "o3003030e00e";
+    public static final String LOGIN_FIRST = "dfsd";
+    public static final String LOGIN_SECOND = "alksdlksdlk@lkksdlklf.llff";
+    public static final String LOGIN_THIRD = "eoooooppepep";
+    public static final String LOGIN_FOURTH = "osoooeoeoe";
+    public static final String LOGIN_FIFTH = "dfghjkfghj";
+    public static final String LOGIN_SIXTH = "dkdkkd";
+    public static final String LOGIN_SEVENTH = "d23232kdkkd";
+    public static final int AGE_FIRST = 16;
+    public static final int AGE_SECOND = 18;
+    public static final int AGE_THIRD = -23939;
+    public static final int AGE_FOURTH = 999;
+    public static final int AGE_INITIAL = 90;
```

Comment:
a lot of unnecessary constants. My suggestion - create a constant only if you use the same constant 2+ times in different tests.